### ISSUE TITLE
noise_vs_tone (and other functions?) using odd default args

### ISSUE THREAD

**Issue by swh76** [slaclab/pysmurf#355]
`noise_vs_tone` and probably other functions are using some very specific and dumb defaults ; e.g. fraction_full_scale=.72 :

https://github.com/slaclab/pysmurf/blob/6a35876e369d227e907b7df3f95655473749060f/python/pysmurf/client/debug/smurf_noise.py#L330

it should instead have the default fraction_full_scale=None, which then pulls whatever's in the config or stored at runtime from the last `tracking_setup` call.  Whoever closes this issue should scan over pysmurf functions to look for functions with similarly bad defaults.

**Comment by eyyoung24** [comment]
Do you want to move things like `meas_time` defaults into configs too? It makes it harder for the user to use the `?` and `??` function in ipython to see what the defaults are. Obviously we could write the default value in the doc string, but that's hard to keep up with.

**Comment by swh76** [comment]
Good point @eyyoung24, it's more subjective than I initially thought.  I guess I'd vote for n_phi0 being in the cfg and meas_time not being in the cfg?  What do you think?
